src/prelude.rs

```diff
@@ -1,12 +1,5 @@
+#[doc(no_inline)]
```

Comment:
The `search` module currently isn't included in this "kitchen sink" prelude. It could be with conditional compilation?

Reply:
Done